File: 2022.07.14/양지원/SharedPreferences/app/src/main/java/com/example/sharedpreferences/MainActivity.java
package com.example.sharedpreferences;

import androidx.appcompat.app.AppCompatActivity;
import android.app.AlertDialog;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.Button;
import android.widget.CheckBox;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //체크박스를 체크했는지 유무 불러온다.
        ShowCheck shared = new ShowCheck();
        Boolean flag = shared.getFirstLogin(this);

        //체크하지 않았으면 다이얼로그를 실행한다.
        if(!flag){
            AlertDialog.Builder builder = new AlertDialog.Builder (this);
            View view = LayoutInflater.from (this).inflate (R.layout.dialog, null, false);
            builder.setView (view);
            final AlertDialog dialog = builder.create ();
            final CheckBox cb = view.findViewById (R.id.checkBox);
            final Button button = view.findViewById (R.id.btn_close);

            button.setOnClickListener (v -> {
                //체크박스 체크했으면 true 를 저장한다.
                if (cb.isChecked ()) {
                    shared.setFirstLogin (this, cb.isChecked ());
                }
                dialog.dismiss ();
            });
            dialog.show ();
        }
    }
}

File: 2022.09.15/김준호/OX/app/src/main/java/com/example/ox/MainActivity.kt
package com.example.ox

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.Editable
import android.text.TextWatcher
import android.widget.ImageButton
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var clickCount = 0
        var answer = 0

        val answerList = listOf<Int>(1,1,0,1,0,0,0,0,1,1,1,1)

        val btnR: ImageButton = findViewById(R.id.right)
        val btnL: ImageButton = findViewById(R.id.left)
        val tv: TextView = findViewById(R.id.tv)

        btnL.setOnClickListener {
            if(clickCount < 12){
                if(answerList[clickCount] == 0){
                    answer++
                }
                clickCount++
                tv.text = clickCount.toString()

            }

        }

        btnR.setOnClickListener {














            if(clickCount < 12){
                if(answerList[clickCount] == 1){
                    answer++
                }

                clickCount++
                tv.text = clickCount.toString()

            }
        }

        tv.addTextChangedListener(object :TextWatcher{
            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {}

            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
                if(clickCount == 12){
                    val intent = Intent(this@MainActivity,AnswerActivity::class.java)
                    intent.putExtra("answer",answer)
                    startActivity(intent)
                }
            }

            override fun afterTextChanged(p0: Editable?) {}

        })
    }
}